.github/ISSUE_TEMPLATE/property-request-form.yml

```diff
@@ -32,12 +32,39 @@ body:
         - Deals
         - Tickets
         - Other
+  - type: checkboxes
+    id: company-type
+    attributes:
+      label: If the Object Association is Company: Which company type does this information apply to?
```

Comment:
This request form will produce an error because you can't have a second `:` as you already have `label:`. You might want to `If you choose Company in the question above, which company type does this information apply to?`